website/components/AppWrapper.tsx

```diff
@@ -56,20 +59,41 @@ export default function AppWrapper(inputProps: AppWrapperProps): ReactElement {
     // On load, fetch parameters from the URL and location state, then merge.
     const locationArgs = location.state as AppDataProps;
     getArgsFromParams(searchParams).then(({ args: urlArgs, viewerSettings: urlViewerSettings }) => {
-      setViewerArgs({ ...DEFAULT_APP_PROPS, ...locationArgs, ...urlArgs });
+      setViewerArgs({ ...DEFAULT_APP_PROPS, ...urlArgs, ...locationArgs });
```

Comment:
Location-based arguments (ones given through React router) take precedence over the URL now. This shouldn't make a difference in practice, but means that internal routing (through loading/load buttons) is guaranteed to have control of what data is being loaded.